crates/biome_js_parser/src/syntax/typescript/statement.rs

```diff
@@ -241,8 +243,8 @@ pub(crate) fn parse_ts_declare_statement(p: &mut JsParser) -> ParsedSyntax {
         // test_err ts ts_declare_const_initializer
         // declare @decorator class D {}
         // declare @decorator abstract class D {}
-        parse_declaration_clause(p, stmt_start_pos)
-            .expect("Expected a declaration as guaranteed by is_at_ts_declare_statement")
+        // Expected a declaration as guaranteed by is_at_ts_declare_statement
```

Comment:
I think this comment is redundant now, because it doesn't guarantee

Reply:
yeah, i will remove it~